framework/src/org/checkerframework/common/value/ReflectiveEvaluator.java

```diff
@@ -94,6 +124,36 @@ public ReflectiveEvalutator(
     }
 
     /**
+     * This method normalizes an array of arguments to a varargs method by changing the arguments
+     * associated with the varargs parameter into an array.
+     *
+     * @param arguments an array of arguments for {@code method}. The length is at least {@code
+     *     method.numberOfParameters-1}.
+     * @param method a method whose last formal parameter is a varargs parameter
+     * @return the length of the array is exactly {@code method.numberOfParameters-1}
```

Comment:
Shouldn't this be method.numberOfParameters (without "-1")?

I agree that this is clearer as a method that works on one "invocation" at a time.